js/ui/lookup.js

```diff
@@ -977,12 +968,13 @@ var Lookup = DropDownList.inherit({
             var currentDevice = devices.current(),
                 searchMode = currentDevice.android && currentDevice.version[0] >= 5 ? "text" : "search";
 
+            let isKeyboardListeningEnabled = false;
+
             this._searchBox = this._createComponent($searchBox, TextBox, {
-                onDisposing: function() {
-                    this._detachSearchChildProcessor();
-                }.bind(this),
-                onFocusIn: this._searchFocusHandler.bind(this),
-                onFocusOut: this._searchBlurHandler.bind(this),
+                onDisposing: () => isKeyboardListeningEnabled = false,
+                onFocusIn: () => isKeyboardListeningEnabled = true,
+                onFocusOut: () => isKeyboardListeningEnabled = false,
```

Comment:
Should we create a class property and change it by methods instead of variables?

Reply:
Why?